org-formation/050-costs/categories.yaml

```diff
@@ -28,151 +28,7 @@ Resources:
-              - "975466401039"
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "122000"
-          - Value: '101600 NIH ITCR'
-            Accounts:
-              - "216152803258"
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "101600"
-          - Value: '112501 Mobile Toolbox Project Core'
-            Accounts:
-              - "634761300905"
-              - "611413694531"
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "112501"
-          - Value: '314900 iAtlas'
-            Accounts:
-              - "386990716034"
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "314900"
-          - Value: '120100 HTAN DFCI'
-            Accounts:
-              - "888810830951"
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "120100"
-          - Value: '101500 NIA AMP AD CC'
-            Accounts:
-              - "607346494281"
-              - "681175625864"
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "101500"
-          - Value: '101700 NIA AMP AD Supp Multi Scale'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "101700"
-          - Value: '106000 UMass PECDCC'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "106000"
-          - Value: '122500 MGH RECOVER'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "122500"
-          - Value: '312000 Genie AACR'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "312000"
-          - Value: '312100 Genie AACR BPC'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "312100"
-          - Value: '315200 Wellcome Leap 1kd'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "315200"
-          - Value: '40033 Genentech imCore'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "40033"
-          - Value: '000000 NO PROGRAM'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "000000"
-        InheritedValues:
-          TagOrder:
-            - CostCenterOther
-            - CostCenter
-          RulePosition: Last
+      'Fn::Transform':
```

Comment:
hmm, i must have missed it but why did you include `RuleVersion` into program-code-rules.yaml?  Wouldn't it be better to keep them separate so that, if you ever choose, you can set them independently?

Reply:
i don't think it makes sense to ever set them separately, `RuleVersion` is setting the schema used by `Rules`

Reply:
There are other properties in `AWS::CE::CostCategory`.  This implementation  suggests that the lambda will provide all of the properties for the module correct? What if you wanted to configure other parameters for this CostCategory module?  Or what if parameters get added/removed from it?

Reply:
no, my intention isn't to provide all of the properties. originally the lambda was only going to manage the value of the `Rules` property, but the `Rules` and `RuleVersion` properties are so tightly coupled that I decided to implement both. though I don't see a need to implement any other properties

and I'm not sure exactly what you mean by parameters getting added/removed, but if cloudformation makes a breaking change to a resource schema, then that would force us to update our templates no matter what.

Reply:
No not necessarily, it depends on what the parameter does. If the parameter is tightly coupled to the rules generated then it would have to go in the lambda, but if it's an independent configuration I would prefer to set it in org-formation alongside where we set the category `Name`.

Reply:
we should discuss, I think we might have a different understand of how this works.